packages/trpc/server/routers/viewer/_router.tsx

```diff
@@ -84,4 +86,6 @@ export const viewerRouter = router({
   credits: creditsRouter,
   ooo: oooRouter,
   travelSchedules: travelSchedulesRouter,
+  ai: aiRouter,
```

Comment:
AI seems too generic. Cal.com can use AI for anything, how about we call calAiPhone only